### PR TITLE
12.0 isabel currency

### PR DESCRIPTION
**Pull request by IsabelEsparza**
@alan196 could you check it please? 

Thank You!

### CODE REVIEW

File: purchase_supplierinfo_currency/models/product.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+
+class ProductProduct(models.Model):
+    _inherit = 'product.product'
+
+    def _prepare_sellers(self, params):
+        res = super()._prepare_sellers(params)
+        if not params:
+            return res
+        if params.get('supplier'):
```
Comment (nikul-serpentcs) [inline purchase_supplierinfo_currency/models/product.py:15]: @IsabelEsparza Here use `Else`

```
if params.get('supplier'):
    currency_id = params.get('supplier').currency_id.id
else:
    currency_id = params.get('order_id').partner_id.currency_id.id
```

**Review by nikul-serpentcs — CHANGES_REQUESTED** [review]
Improve Code

File: purchase_supplierinfo_currency/models/product.py (hunk 1 above)
Comment (IsabelEsparza) [reply purchase_supplierinfo_currency/models/product.py:15]: Thank you, 
I change it right now.

**Review by alan196 — APPROVED** [review]
👍 Tested functionally and code review

**Pull request closed without merging**